.circleci/config.yml

```diff
@@ -40,7 +40,7 @@ jobs:
           name: Build and deploy to Testflight
           command: bundle exec fastlane testflight_beta
 
-  build-and-deploy-appcenter:
+  build-and-deploy-appcenter-and-browserstack:
```

Comment:
Personally I'd say this is more organised if split into two separate jobs that run in the same workflow consecutively or in parallel